### ISSUE TITLE
Review

### ISSUE THREAD

**Issue by Giorgio-Galasso** [MarilenaDel/CI2024_lab1#1]
Hi Marilena,
I read your README. First of all, I wanted to tell you that you did a great job. Nowadays, it's rare to start from scratch for any code, so your solution is completely valid.
You used a Tabu Search algorithm that is based on the idea of intelligently exploring the solution space while avoiding revisiting already explored solutions. However, Tabu Search may require a high number of iterations, especially in the later cases where the numbers are larger.

The observation made about the length of the tabu list depending on the various analyzed cases is interesting.

For future labs, it could be useful for both you and the reviewers to include graphs like those we saw in class.

In conclusion, I have nothing else to add. The adaptation of the algorithm to the problem at hand is commendable and appears to be well-executed. Well done! 💪
